docs/documentation/functions/aria-suite/aria-suite-lifecycle/Get-vRSLCMProductDetails.md

```diff
@@ -0,0 +1,47 @@
+# Get-vRSLCMProductDetails
+
+## SYNOPSIS
+
+Get Product Details in VMware Aria Suite Lifecycle.
+
+## SYNTAX
+
+```powershell
+Get-vRSLCMProductDetails [-productid] <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-vRSLCMProductDetails` cmdlet gets the product details for a certain product in VMware Aria Suite Lifecycle.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Get-vRSLCMProductDetails -productid vrli
+```
+
+This example gets all environments in VMware Aria Suite Lifecycle.
+
+## PARAMETERS
+
+### -productid
+
+The product ID of the product to get the details for.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).

```

Comment:
```suggestion
```suggestion
This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, -`InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).

```
```